### ISSUE TITLE
The signature field in the second parameter info of the signAsync function in PdfSigner does not support Chinese. Can you help me take a look

### ISSUE THREAD

**Issue by BQ97** [remdra/sign-pdf-lib#4]
```ts
const info = {
	pageNumber: 1,

	name: 'Signature99',

	signature: {
		name: '小白',
		location: '哈哈哈',
		reason:'我同意',
		date: new Date(),
	},

	visual: {
		rectangle: {
			left: 50,
			top: 241,
			right: 164,
			bottom: 411,
		},
		background: fs.readFileSync('./sign-pdf-lib/sign.jpg'),
	},
};

const settings = {
	signatureLength: 4000 - 6,
	rangePlaceHolder: 9999999,
	signatureComputer: {
		certificate: fs.readFileSync('./sign-pdf-lib/cert.p12'),
	},
};
const filePath = './sign-pdf-lib/test.pdf';

const pdfSigner = new PdfSigner(settings);

const fileBytes = fs.readFileSync(filePath);

const signedPdf = await pdfSigner.signAsync(fileBytes, info);
```
After the signature field above is written in Chinese, the Name, Location, and Reason in the signature field of the generated PDF are all garbled

**Comment by remdra** [comment]
Please provide me a file signed with SAME information. That way I'll be able to see what goes wrong.

**Comment by BQ97** [comment]
I have found the reason. It turns out that PDF lib does not support Unicode. You can use the PDFString. of() function in PDF lib to pass in a Unicode encoding and see the effect. For example, if you pass in \ u6211, it will eventually cut out 62 and leave only 11. We need to find a way to make our class library support Unicode

**Comment by remdra** [comment]
I think I have a solution, a fix will be soonnavailable.

**Comment by remdra** [comment]
fixed in version 2.0.1

**Comment by BQ97** [comment]
Okay, thank you very much. I'll go test it now
